src/lib/CryptoManager.js

```diff
@@ -119,19 +117,25 @@ class CryptoManager {
   };
 
   getRandomBytes = (size: number): Promise<Buffer> =>
-    new Promise((resolve, reject) => {
-      crypto.randomBytes(size, (error, buffer) => {
-        if (error) {
-          reject(error);
-          return;
-        }
-
-        resolve(buffer);
-      });
+    new Promise((
+      resolve: (buffer: Buffer) => void,
+      reject: (error: Error) => void,
+    ) => {
+      crypto.randomBytes(
+        size,
+        (error: ?Error, buffer: Buffer) => {
+          if (error) {
+            reject(error);
+            return;
+          }
+
+          resolve(buffer);
+        },
+      );
     });
 
   static getRandomUINT16 = (): number => {
-    const uintMax = Math.pow(2, 16) - 1;
+    const uintMax = 2 ** 16 - 1;
```

Comment:
Ok, please add a comment on what you're doing. Not many people will know about this operator.